x-pack/plugins/integration_assistant/server/integration_builder/fields.ts

```diff
@@ -6,35 +6,48 @@
  */
 
 import nunjucks from 'nunjucks';
-
+import { load } from 'js-yaml';
```

Comment:
`safeLoad` instead?

Reply:
No, `import { load }` is the correct way since the PR https://github.com/elastic/kibana/pull/190678 has been merged